components/azureblob/azureblob.wick

```diff
@@ -0,0 +1,48 @@
+kind: wick/component@v1
+name: azureblob
+metadata:
+  version: 0.0.1
+  description: New composite wick component
+  licenses:
+    - Apache-2.0
+import:
+  - name: azureblob
+    component:
+      kind: wick/component/manifest@v1
+      ref: ./azureblob-http-client.wick
+      with:
+        storage_account: '{{ ctx.root_config.storage_account }}'
+        container_name: '{{ ctx.root_config.container_name }}'
+  - name: hmac
+    component:
+      kind: wick/component/manifest@v1
+      ref: registry.candle.dev/common/hmac:0.0.1
+      with:
+        secret: '{{ ctx.root_config.sas_token }}'
+component:
+  kind: wick/component/composite@v1
+  with:
+    - name: storage_account
+      type: string
+    - name: container_name
+      type: string
+    - name: sas_token
```

Comment:
```suggestion
    - name: access_key
```

its not a SAS token.  That is something different. please change this to 'access_key'